components/x-audio/src/components/styles.scss

```diff
@@ -45,8 +46,8 @@ $image-size: 72px;
 	grid-template-areas: ". minimise-button ."
 	"title title title"
 	"series-name series-name series-name"
-	"control-timeline control-timeline control-timeline"
-	"status . remaining"
+	"timeline timeline timeline"
+	"timeline timeline timeline"
```

Comment:
if `min-content` deleted from this line like this
https://github.com/Financial-Times/x-dash/pull/334/files#diff-9e5dcc8f41580bd73ebe25d2fa925339L44
=> `grid-template-rows: 48px ($image-size/2) ($image-size/2) auto 96px 72px;`
we could do `"timeline timeline timeline"` only one line.